atef/tests/test_widgets.py

```diff
@@ -210,13 +215,13 @@ def test_config_window_save_load(qtbot: QtBot, tmp_path: pathlib.Path):
         assert source_lines == dest_lines
 
 
-def test_edit_run_toggle(qtbot: QtBot, test_configs: list[os.PathLike]):
+@pytest.mark.skip(reason='inconsistent segfaults on CI, passes locally, (04/17/23)')
```

Comment:
Something I've been meaning to figure out: is there a way to skip tests _only_ on CI? I think in some cases this would be helpful so as to not need to edit the local test suite to run tests that inexplicably brick on GHA.

Maybe the easiest way would be to check for the `CI` environment variable, or maybe there's something better?
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Seems doable 👀

Reply:
It's worth being cautious here and not getting ahead of ourselves to point the finger at CI.

I think there are some threading-related crashes that are the result of actual flaws

Reply:
Test has been re-enabled, it passes more consistently now though I don't know exactly what helped it get there